### PR TITLE
Cut v2.40.0-rc.0

### PR DESCRIPTION
**Pull request by codesome**
This is currently pointing to `main` branch, but I will create a new `release-2.40` branch once https://github.com/prometheus/prometheus/pull/11524 is merged. PR is anyway ready for review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2.40.0-rc.0 / 2022-11-03
+
+This release introduces an experimental, native way of representing and storing histograms.
+It can be enabled in Prometheus via `--enable-feature=native-histograms` to accept native histograms.
+Use the `main` branch of `client_golang` to instrument your application with native histograms (this will change for the final release when v1.14.0 of client_golang will be out).
```
Comment (codesome) [inline CHANGELOG.md:7]: This messaging is temporary for the release candidate. For the final release, (1) v1.14.0 of client_golang will be out with native histogram support, which I will mention here (2) I will be writing a blog post for prometheus.io blog introducing native histograms and explaining how to use it, and I will link that here as well.

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
Please update UI as well

File: CHANGELOG.md (hunk 1 above)
Comment (beorn7) [reply CHANGELOG.md:7]: You also have to change an option in your instrumentation.

In the way it is worded now, it sounds a bit like all histograms will change to native histograms the moment you update client_golang and set the feature flag.

**Review by beorn7 — COMMENTED** [review]
Maybe also mention that setting the feature flag for native histograms switches the preferred exposition format to protobuf.

**Comment by beorn7** [comment]
> Please update UI as well

Shouldn't that have happened already via dependabot? @Nexucis do you know what the current state is?

**Comment by codesome** [comment]
Oh right. I forgot what I did last time and thought it is done after the release. My memory is not serving me well.

File: CHANGELOG.md (hunk 1 above)
Comment (codesome) [reply CHANGELOG.md]: @beorn7 how does it sound now?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+This release introduces an experimental, native way of representing and storing histograms.
+
+It can be enabled in Prometheus via `--enable-feature=native-histograms` to accept native histograms.
+Enabling native histograms will also switch the preferred exposition format to protobuf.
+
+To instrument your application with native histograms, use the `main` branch of `client_golang` (this will change for the final release when v1.14.0 of client_golang will be out), and set the `NativeHistogramBucketFactor` in your `HistogramOpts` (`1.1` is a good starting point).
+Your existing histograms won't switch to native histograms until `NativeHistogramBucketFactor` is not set.
+
+* [FEATURE] Add **experimental** support for native histograms. Enable with the flag `--enable-feature=native-histograms`. #11447
+* [FEATURE] SD: Add service discovery for OVHcloud. #10802
```
Comment (codesome) [inline CHANGELOG.md:14]: @roidelapluie can you confirm this? I see that it got merged few hours ago

Comment (roidelapluie) [reply CHANGELOG.md:14]: Yessss :)

**Comment by codesome** [comment]
Merging this to tag the release. If there is any more feedback later for the changelog message, I will open another PR and take care of it manually when publishing the release.

**Comment by codesome** [comment]
Benchmark looks good so far. Memory, CPU, query times, are all similar.

**Comment by codesome** [comment]
Benchmarks looked good until the end.

**Pull request merged**